static/app/components/tabs/index.stories.tsx

```diff
@@ -188,43 +190,93 @@ export default storyBook(Tabs, story => {
     </SideBySide>
   ));
 
-  story('Variants', () => (
-    <div>
-      <p>
-        Use the variant prop to control which tab design to use. The default, "flat", is
-        used in the above examples, but you can also use "filled" variant, as shown below.
-        Note that the "filled" variant does not work when the oritentation is vertical
-      </p>
-      <SizingWindow>
-        <Tabs>
-          <TabList variant={'filled'}>
-            {TABS.map(tab => (
-              <TabList.Item key={tab.key}>{tab.label}</TabList.Item>
-            ))}
-          </TabList>
-          <TabPanels>
-            {TABS.map(tab => (
-              <TabPanels.Item key={tab.key}>{tab.content}</TabPanels.Item>
-            ))}
-          </TabPanels>
-        </Tabs>
-      </SizingWindow>
-      <br />
-      <p>You can also use the "floating" variant, which is used below</p>
-      <SizingWindow>
-        <Tabs>
-          <TabList variant="floating" hideBorder>
-            {TABS.map(tab => (
-              <TabList.Item key={tab.key}>{tab.label}</TabList.Item>
-            ))}
-          </TabList>
-          <TabPanels>
-            {TABS.map(tab => (
-              <TabPanels.Item key={tab.key}>{tab.content}</TabPanels.Item>
-            ))}
-          </TabPanels>
-        </Tabs>
-      </SizingWindow>
-    </div>
-  ));
+  story('Variants', () => {
+    const propMatrix: {
+      hideBorder: (boolean | undefined)[];
+      orientation: (Orientation | undefined)[];
+      variant: (BaseTabProps['variant'] | undefined)[];
+    } = {
```

Comment:
This is possible to type with something like: `const propMatrix: PropMatrix<BaseTabProps> = { ... }`
or whatever the component props are.